install/operator/build/conf/config.yaml

```diff
@@ -5,6 +5,7 @@ EarlyAccess: false
 Images:
   ImageStreamNamespace: ""
   KomodoImagesPrefix: teiid
+  PostgresqlImagesPrefix: rhscl
```

Comment:
just for the sake of consistency, `PostgresqlImage(s)Prefix` -> `PostgresqlImagePrefix`

Reply:
Solved with https://github.com/syndesisio/syndesis/pull/7255/commits/f1a3e8b10d5f8de930ff1bd5a26cb780f43368e7